checksum.go

```diff
@@ -20,7 +22,7 @@ func verifyChecksumOfReleaseAsset(assetPath string, checksumMap map[string]bool,
 		keys := reflect.ValueOf(checksumMap).MapKeys()
 		return newError(checksumDoesNotMatch, fmt.Sprintf("Expected to checksum value to be one of %s, but instead got %s for Release Asset at %s. This means that either you are using the wrong checksum value in your call to fetch, (e.g. did you update the version of the module you're installing but not the checksum?) or that someone has replaced the asset with a potentially dangerous one and you should be very careful about proceeding.", keys, computedChecksum, assetPath))
 	}
-	fmt.Printf("Release asset checksum verified for %s\n", assetPath)
+	logger.Printf("Release asset checksum verified for %s\n", assetPath)
```

Comment:
If we have a proper logger now, we should switch from `Printf` to `Infof`, `Debugf`, `Errorf`, etc, as appropriate.

Reply:
@brikis98 Good point, then I vote `Infof` for most of these calls as users will have come to expect them. Thoughts?

Reply:
Use your judgment!

Reply:
Moving to `Infof`